### DISCUSSION TITLE
useFetch docs sample not working

### DISCUSSION THREAD

**Discussion by lucianobosco** [nuxt/nuxt#16683] [Questions]
First day with nuxt. I'm trying to replicate the nuxt 3 `useFetch` sample but no luck so far.
My `post.vue` page contains the below code:

```javascript
<template>
  <section>
    {{data}}
  </section>
</template>

<script setup>
  const { data, pending, error, refresh } = await useFetch('https://api.nuxtjs.dev/mountains',{
    pick: ['title']
  })

  console.log(data.value)
</script>
```
For some reason the data is not shown in template, just `{}`.
The `console.log()` shows `Proxy {title: undefined}`

I've realized that removing the `pick` option solves the issue, so I'm wondering if `pick` only works for objects and can't be used in arrays.
It's odd because the sample is using an array https://v3.nuxtjs.org/api/composables/use-fetch

Any advice will be appreciated

**Comment by fwdmo** [comment]
I have a similar issue, the data fetched not rendered but in in when put it in console.log and open devtools i can see it
